apps/challenges/models.py

```diff
@@ -271,17 +280,30 @@ def is_active(self):
     def save(self, *args, **kwargs):
 
         # If the max_submissions_per_day is less than the max_concurrent_submissions_allowed.
-        if (
-            self.max_submissions_per_day
-            < self.max_concurrent_submissions_allowed
-        ):
-            self.max_concurrent_submissions_allowed = (
-                self.max_submissions_per_day
-            )
-
-        challenge_phase_instance = super(ChallengePhase, self).save(
-            *args, **kwargs
-        )
+        if self.max_submissions_per_day < self.max_concurrent_submissions_allowed:
+            self.max_concurrent_submissions_allowed = self.max_submissions_per_day
+
+        # If challengephase start datetime is less than challenge start datetime.
```

Comment:
can you change `challengephase` to `ChallengePhase`

Reply:
@Ram81 Sure